contracts/votingMachines/GenesisProtocolLogic.sol

```diff
@@ -224,14 +229,16 @@ contract GenesisProtocolLogic is IntVoteInterface {
         require(proposal.state == ProposalState.Boosted || proposal.state == ProposalState.QuietEndingPeriod,
         "proposal state in not Boosted nor QuietEndingPeriod");
         require(_execute(_proposalId), "proposal need to expire");
-        uint256 expirationCallBountyPercentage =
+        uint256 blocksSinceTimeOut =
         // solhint-disable-next-line not-rely-on-time
-        (uint(1).add(now.sub(proposal.currentBoostedVotePeriodLimit.add(proposal.times[1])).div(15)));
-        if (expirationCallBountyPercentage > 100) {
-            expirationCallBountyPercentage = 100;
+            now.sub(proposal.currentBoostedVotePeriodLimit.add(proposal.times[1])).div(15);
+
+        if (blocksSinceTimeOut > 100) {
+            blocksSinceTimeOut = 100;
         }
-        proposal.expirationCallBountyPercentage = expirationCallBountyPercentage;
-        expirationCallBounty = expirationCallBountyPercentage.mul(proposal.stakes[YES]).div(100);
+
+        proposal.expirationCallBountyPercentage = blocksSinceTimeOut;
```

Comment:
you write in the docs that the maximum is 10%, but this goes up to 100, it seems to me.

Reply:
I see that calcExecuteCallBounty divides by 1000, but please please please do not have a variable called PerCentage that stores ProMille.

Reply:
> you write in the docs that the maximum is 10%, but this goes up to 100, it seems t

yes, 100 is 10%  - 
see https://github.com/daostack/infra/blob/execute_boosted/contracts/votingMachines/GenesisProtocolLogic.sol#L76 

and 

https://github.com/daostack/infra/blob/execute_boosted/contracts/votingMachines/GenesisProtocolLogic.sol#L442

Reply:
I know, my objection is against the variable name, not the calculation :-)

Reply:
just call it  `expirationCallBountyProMille` and everything is fine :-)

Reply:
see 
0720cf1